src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -76,6 +76,7 @@
     <InternalsVisibleTo Include="Roslyn.VisualStudio.Setup" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.VisualBasic.Repl" />
     <InternalsVisibleTo Include="Roslyn.VisualStudio.DiagnosticsWindow" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Remote.ServiceHub" />
```

Comment:
this won't matter since OOP won't reference editor layer. service that doesn't require editor dependency should be in feature layer.